2021-12-23 11:43:37 INFO     Waiting 60 seconds for internet connection and influxDB to establish
2021-12-23 11:43:42 INFO     Setup InfluxDB Client... 
2021-12-23 11:43:45 INFO     Internet Connection Found...Enabling Cloud Write if Set.
2021-12-23 11:43:45 ERROR    [Errno 2] could not open port /dev/ttyUSB0: [Errno 2] No such file or directory: '/dev/ttyUSB0'
2021-12-23 11:43:45 INFO     Loading inverters... 
2021-12-23 11:43:45 ERROR    Error Loading Inverter Info.  Error Count 1
2021-12-23 11:44:00 ERROR    [Errno 2] could not open port /dev/ttyUSB0: [Errno 2] No such file or directory: '/dev/ttyUSB0'
2021-12-23 11:44:00 INFO     Loading inverters... 
2021-12-23 11:44:00 ERROR    Error Loading Inverter Info.  Error Count 2
2021-12-23 11:44:15 ERROR    [Errno 2] could not open port /dev/ttyUSB0: [Errno 2] No such file or directory: '/dev/ttyUSB0'
2021-12-23 11:44:15 INFO     Loading inverters... 
2021-12-23 11:44:15 ERROR    Error Loading Inverter Info.  Error Count 3
2021-12-23 11:44:30 ERROR    [Errno 2] could not open port /dev/ttyUSB0: [Errno 2] No such file or directory: '/dev/ttyUSB0'
2021-12-23 11:44:30 INFO     Loading inverters... 
2021-12-23 11:44:30 ERROR    Error Loading Inverter Info.  Error Count 4
2021-12-23 11:44:45 ERROR    [Errno 2] could not open port /dev/ttyUSB0: [Errno 2] No such file or directory: '/dev/ttyUSB0'
2021-12-23 11:44:45 INFO     Loading inverters... 
2021-12-23 11:44:45 ERROR    Error Loading Inverter Info.  Error Count 5
2021-12-23 11:45:00 ERROR    [Errno 2] could not open port /dev/ttyUSB0: [Errno 2] No such file or directory: '/dev/ttyUSB0'
2021-12-23 11:45:00 INFO     Loading inverters... 
2021-12-23 11:45:00 ERROR    Error Loading Inverter Info.  Error Count 6
2021-12-23 11:45:15 ERROR    [Errno 2] could not open port /dev/ttyUSB0: [Errno 2] No such file or directory: '/dev/ttyUSB0'
2021-12-23 11:45:15 INFO     Loading inverters... 
2021-12-23 11:45:15 ERROR    Error Loading Inverter Info.  Error Count 7
2021-12-23 11:45:30 ERROR    [Errno 2] could not open port /dev/ttyUSB0: [Errno 2] No such file or directory: '/dev/ttyUSB0'
2021-12-23 11:45:30 INFO     Loading inverters... 
2021-12-23 11:45:30 ERROR    Error Loading Inverter Info.  Error Count 8
2021-12-23 11:45:45 ERROR    [Errno 2] could not open port /dev/ttyUSB0: [Errno 2] No such file or directory: '/dev/ttyUSB0'
2021-12-23 11:45:45 INFO     Loading inverters... 
2021-12-23 11:45:45 ERROR    Error Loading Inverter Info.  Error Count 9
2021-12-23 11:46:00 ERROR    [Errno 2] could not open port /dev/ttyUSB0: [Errno 2] No such file or directory: '/dev/ttyUSB0'
2021-12-23 11:46:00 INFO     Loading inverters... 
2021-12-23 11:46:00 ERROR    Error Loading Inverter Info.  Error Count 10
2021-12-23 11:46:15 INFO     reached max error tolerance.  Rebooting Pi.
2021-12-23 11:46:48 INFO     Waiting 60 seconds for internet connection and influxDB to establish
2021-12-23 11:46:53 INFO     Setup InfluxDB Client... 
2021-12-23 11:47:34 INFO     Internet Connection Found...Enabling Cloud Write if Set.
2021-12-23 11:47:34 ERROR    [Errno 2] could not open port /dev/ttyUSB0: [Errno 2] No such file or directory: '/dev/ttyUSB0'
2021-12-23 11:47:34 INFO     Loading inverters... 
2021-12-23 11:47:34 ERROR    Error Loading Inverter Info.  Error Count 1
2021-12-23 11:47:49 ERROR    [Errno 2] could not open port /dev/ttyUSB0: [Errno 2] No such file or directory: '/dev/ttyUSB0'
2021-12-23 11:47:49 INFO     Loading inverters... 
2021-12-23 11:47:49 ERROR    Error Loading Inverter Info.  Error Count 2
2021-12-23 11:48:04 ERROR    [Errno 2] could not open port /dev/ttyUSB0: [Errno 2] No such file or directory: '/dev/ttyUSB0'
2021-12-23 11:48:04 INFO     Loading inverters... 
2021-12-23 11:48:04 ERROR    Error Loading Inverter Info.  Error Count 3
2021-12-23 11:48:19 ERROR    [Errno 2] could not open port /dev/ttyUSB0: [Errno 2] No such file or directory: '/dev/ttyUSB0'
2021-12-23 11:48:19 INFO     Loading inverters... 
2021-12-23 11:48:19 ERROR    Error Loading Inverter Info.  Error Count 4
2021-12-23 11:48:35 ERROR    [Errno 2] could not open port /dev/ttyUSB0: [Errno 2] No such file or directory: '/dev/ttyUSB0'
2021-12-23 11:48:35 INFO     Loading inverters... 
2021-12-23 11:48:35 ERROR    Error Loading Inverter Info.  Error Count 5
2021-12-23 11:48:50 ERROR    [Errno 2] could not open port /dev/ttyUSB0: [Errno 2] No such file or directory: '/dev/ttyUSB0'
2021-12-23 11:48:50 INFO     Loading inverters... 
2021-12-23 11:48:50 ERROR    Error Loading Inverter Info.  Error Count 6
2021-12-23 11:49:05 ERROR    [Errno 2] could not open port /dev/ttyUSB0: [Errno 2] No such file or directory: '/dev/ttyUSB0'
2021-12-23 11:49:05 INFO     Loading inverters... 
2021-12-23 11:49:05 ERROR    Error Loading Inverter Info.  Error Count 7
2021-12-23 11:49:20 ERROR    [Errno 2] could not open port /dev/ttyUSB0: [Errno 2] No such file or directory: '/dev/ttyUSB0'
2021-12-23 11:49:20 INFO     Loading inverters... 
2021-12-23 11:49:20 ERROR    Error Loading Inverter Info.  Error Count 8
2021-12-23 11:49:35 ERROR    [Errno 2] could not open port /dev/ttyUSB0: [Errno 2] No such file or directory: '/dev/ttyUSB0'
2021-12-23 11:49:35 INFO     Loading inverters... 
2021-12-23 11:49:35 ERROR    Error Loading Inverter Info.  Error Count 9
2021-12-23 11:49:50 ERROR    [Errno 2] could not open port /dev/ttyUSB0: [Errno 2] No such file or directory: '/dev/ttyUSB0'
2021-12-23 11:49:50 INFO     Loading inverters... 
2021-12-23 11:49:50 ERROR    Error Loading Inverter Info.  Error Count 10
2021-12-23 11:50:05 INFO     reached max error tolerance.  Rebooting Pi.
2021-12-23 11:50:27 INFO     Waiting 60 seconds for internet connection and influxDB to establish
2021-12-23 11:50:32 INFO     Setup InfluxDB Client... 
